getparty.go

```diff
@@ -332,8 +333,16 @@ func (cmd Cmd) follow(jar http.CookieJar, userUrl string) (session *Session, err
 			jar.SetCookies(u, cookies)
 		}
 	}
-	client := cleanhttp.DefaultClient()
-	client.Jar = jar
+
+	transport, err := cmd.getTransport()
+	if err != nil {
+		return nil, err
+	}
+	client := &http.Client{
+		Transport: transport,
+		Jar:       jar,
+	}
+
 	client.CheckRedirect = func(*http.Request, []*http.Request) error {
```

Comment:
this one could be spliced to above client instantiation.

Reply:
I think I understood - please do double check my commit though :)